internal/controllers/h264_controller.go

```diff
@@ -0,0 +1,59 @@
+package controllers
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/flavioribeiro/donut/internal/entities"
+)
+
+func ParseNALUs(data []byte) (entities.NALUs, error) {
+	var nalus entities.NALUs
+
+	rawNALUs := bytes.Split(data, []byte{0x00, 0x00, 0x01})
+
+	for _, rawNALU := range rawNALUs[1:] {
```

Comment:
I'm new to webRTC. It might be worthy adding a comment explaining why it starts from 1. What is your opinion?

Reply:
It's related to [the h264 bitstream](https://github.com/leandromoreira/donut/blob/main/internal/entities/h264.go#L7C3-L7C35) parsing, maybe add a comment about that. thanks

Reply:
![image](https://github.com/flavioribeiro/donut/assets/55913/0248e883-8dcf-4427-8e61-58692932763e)